src/ir/comp.rs

```diff
@@ -989,6 +943,11 @@ impl CompInfo {
         return self.has_own_virtual_method;
     }
 
+    /// Does this type have a destructor?
+    pub fn has_destructor(&self) -> bool {
+        self.has_destructor
```

Comment:
Let's rename these to either `trivially_has_destructor` or `has_own_destructor` to clarify that this isn't looking through template instantiations or base members.

Reply:
Sure, good idea.

Reply:
Let me know when you update the PR, sometimes I don't get the new-commits-pushed emails for whatever reason. Thanks again!

Reply:
Have done so, but squashed and did a force push. Maybe that's why you didn't get an email?